docs/source/02_get_started/02_setup.md

```diff
@@ -1,120 +1,18 @@
-
-The following command will download a Docker image of around 500Mb in size and initialise a cluster with 2 CPUs and 4Gb
-of RAM, with Docker as the chosen driver.
-
-```bash
-minikube start --cpus 2 --memory 4096 --driver=docker
-```
-If successful, the command above should output the following on your terminal:
-```bash
-...
-🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
-```
-
-Next, we will install `nfs-common` drivers. This is required by the JupyterLab instances, which require NFS mount for 
-`PersistentVolumeClaims` (PVCs). To install it, run:
-
-```shell
-minikube ssh "sudo apt update; sudo apt install nfs-common -y"
-```
-For more details on PVs and PVCs, read the [JupyterHub documentation](https://zero-to-jupyterhub.readthedocs.io/en/latest/jupyterhub/customizing/user-storage.html).
-
-#### MetalLB
-
-[MetalLB](https://metallb.universe.tf/) is the load balancer for bare metal Kubernetes clusters. We will need to configure
-MetalLB to match the QHub configuration. 
-
-Here, we will choose to configure the load balancer to have an IP address of 172.17.10.100 (start) and of 172.17.10.200 (stop). 
-Make sure that the IP range is within the Docker interface subnet, in this case, within the 172.17.0.0/16 range.
- 
-To verify your Docker subnet IP address use:
-```shell
-$ ip route
-```
-Where the output will be something similar to:
-```shell
-default via 192.168.1.1 dev wlp4s0 proto dhcp metric 600 
-172.17.0.0/16 dev docker0 proto kernel scope link src 172.17.0.1 
-```
-This means that you have to ensure that the start/stop IP range for the load balancer is within the 172.17.0.0/16 subnet.
-> NOTE: Your docker subnet may be different from the one shown above.
-
----
-
-You can run MetalLB manually or dynamically by using the Python command below.
-
-To manually configure `metallb` run:
-
-```shell
-minikube addons configure metallb
-```
-
-Minikube does not provide a simple interactive way to configure addons, 
-([as shown in this repository issue](https://github.com/kubernetes/minikube/issues/8283)). Hence, we suggest setting up 
-the configuration using the Python script below with the given values, since there is a DNS name that already points to 
-the address. To do so, paste the bash script below on your terminal.
-
-<details><summary>Click to expand</summary>
-
-```bash
-python <<EOF
-import json
-import os
-
-filename = os.path.expanduser('~/.minikube/profiles/minikube/config.json')
-with open(filename) as f:
-     data = json.load(f)
-
-data['KubernetesConfig']['LoadBalancerStartIP'] = '172.17.10.100'
-data['KubernetesConfig']['LoadBalancerEndIP'] = '172.17.10.200'
-
-with open(filename, 'w') as f:
-     json.dump(data, f)
-EOF
-```
-</details>
-
-If successful, the output should be `✅  metallb was successfully configured`.
-
-Lastly, enable MetalLB by running
-```bash
-minikube addons enable metallb
-```
-To which the output should be `The 'metallb' addon is enabled`.
-
----
```

Comment:
Is this info inlcuded somewhere else in the repo?  I'm assuming so.  If so, then the PR looks good to me.

Reply:
yup, same thoughts here. I assumed that it will be moved to [local tests](https://github.com/laisbsc/qhub-cloud/blob/fix_local_testing_docs/docs/source/06_developers_contrib_guide/04_tests.md)

Reply:
Yup! That is my _pumped up_ version of the team's local testing docs. I have it saved locally and it will part of the next PR updating the docs. I'm a bit rushed with some college duties these last few days but I will get this done soon.
🤞🏽 

Thanks for the review! 🚀